dspace/config/modules/elsevier-sciencedirect.cfg

```diff
@@ -0,0 +1,17 @@
+api.key = ${elsevier.api.key}
+
+api.article.url=http://api.elsevier.com/content/article
+ui.article.url=http://www.sciencedirect.com/science/article
+api.entitlement.url=http://api.elsevier.com/content/article/entitlement
+api.scidir.url=http://api.elsevier.com/content/search/scidir
+entitlement.check.enabled=false
+metadata.field.pii = elsevier.identifier.pii
+metadata.field.doi = dc.identifier
+
+embed.display=false
+embed.display.width=700
+embed.display.height=500
```

Comment:
Please add inline comments to this configuration file to describe all of the above configurations, e.g. what each setting does, possible values (if any), etc. While they obviously also should be documented elsewhere, this is just our best practice for DSpace configurations.